tetrateci/fips.md

```diff
@@ -0,0 +1,98 @@
+## Introduction
+
+Google's BoringCrypto [module][1] is used for [FIPS-compliant Istio builds][2]. BoringCrypto is a core module of the
+BoringSSL library and has been tested by CMVP to be [FIPS validated][3]. Both the Istio control plane and data plane
+are built with these modules. The quickest way to get started with FIPS Istio is to use [our distribution][4].
+
+## FIPS Verification
+
+The easiest way to verify the Go version is with Docker. First, pull the container images from [CloudSmith][5]:
+```shell
+TAG=1.11.4-tetratefips-v0
+$ docker pull containers.istio.tetratelabs.com/operator:$TAG
+$ docker pull containers.istio.tetratelabs.com/pilot:$TAG
+$ docker pull containers.istio.tetratelabs.com/proxyv2:$TAG
+```
+
+Get the operator, pilot, and proxyv2 image digests:
```

Comment:
I feel creating the container and copying the file out and running go version on it is simpler than this procedure? We do have this check @ https://github.com/tetratelabs/istio/blob/fb56abf37fec3a7e53340f6e0ac84c791c93d94e/tetrateci/create_istio_release.sh#L73

Reply:
> We do have this check @

@deva26 are you wanting to mention someone here?

Reply:
@deva26 even if we included the check in our image creation process, this doc is meant for security auditors to perform verification.

Reply:
@danehans - What I meant is to change the doc to reflect the procedure of creating the container and copying the binary instead of the currently described process. WDYT?

Reply:
@deva26 thanks for helping me understand your point. Commit `764f0e8` updates the PR based on your guidance.